packages/auth/src/capabilities.ts

```diff
@@ -1,28 +1,38 @@
-import { adxCapability } from './semantics'
+import { adxCapability, parseAdxResource } from './semantics'
 import * as ucan from './ucans/index'
 
 export const writeCap = (
   did: string,
+  namespace?: string,
   collection?: string,
-  schema?: string,
   record?: string,
 ): ucan.Capability => {
   let resource = did
```

Comment:
I think for capabilities, we should always encode them using the DID. It has a nice semantic simplicity: "I have the authority to write to this repo because I have a token originating from the root DID of the repo". Doesn't require going out to identity providers.

Agreed on needing good terminology here. I like `collection = namespace + dataset` 👍